problem_builder/public/css/questionnaire.css

```diff
@@ -97,6 +97,7 @@
     font-family: FontAwesome;
     font-style: normal;
     text-align: center;
+    font-size: 0;
```

Comment:
I double-checked your other PR and isn't it actually redundant here? We already have `display:none` and you've overridden the font size [here](https://github.com/mckinseyacademy/mcka-theme/pull/227/files#diff-7f8e5c657cdce1929b229a9122c572feR56), so it seems safe to remove it as well.

Reply:
@Agrendalath It's actually a tricky fix. Font awesome icon is rendering in :before element so the font size for the element doesn't matter. That's why I have given 20px font to the :before element.